addons/openebs/template/base/install.sh

```diff
@@ -157,3 +170,10 @@ function openebs_migrate_post_helm_resources() {
     # name changed from openebs-maya-operator > openebs
     kubectl delete clusterrolebinding openebs-maya-operator 2>/dev/null || true
 }
+
+function kubernetes_service_healthy() {
+    local namespace=$1
+    local name=$2
+
+    kubectl -n "$namespace" get endpoints "$name" | grep -v "NAME" | grep -v "<none>" &>/dev/null
+}
```

Comment:
I plan to move this to the common set of functions

Reply:
move this to scripts/common/kuberentes.sh

Reply:
you can omit `grep -v "NAME"` by passing `--no-headers` flag to kubectl